pkg/action/lazyclient.go

```diff
@@ -60,7 +61,7 @@ func newSecretClient(lc *lazyClient) *secretClient {
 	return &secretClient{lazyClient: lc}
 }
 
-func (s *secretClient) Create(ctx context.Context, secret *v1.Secret, opts metav1.CreateOptions) (*v1.Secret, error) {
+func (s *secretClient) Create(ctx context.Context, secret *v1.Secret, opts metav1.CreateOptions) (result *v1.Secret, err error) {
```

Comment:
Why was the return signature changed to `(result *v1.Secret, err error)`?

Reply:
Hi Matt,
This was changed to align with the updated signature in https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/core/v1#SecretInterface